### DISCUSSION TITLE
Using MeiliSearch plugin in Medusa 2.0

### DISCUSSION THREAD

**Discussion by Attachai77** [medusajs/medusa#8404] [Q&A]
Has MeiliSearch plugin suppoted in V2 yet ?

I've installed `"medusa-plugin-meilisearch": "^2.0.11"` in my medusa v2 , so it doesn't sync data to meilisearch.

here is my config.

<img width="761" alt="Screenshot 2567-08-02 at 13 31 12" src="https://github.com/user-attachments/assets/a2ef03c8-9a92-44aa-8c4a-12443e08e8e4">

**Comment by SalahAdDin** [comment]
@Attachai77 Why is this closed?

**Reply by alsherif-khalaf** [reply]
Is it working?

**Reply by SalahAdDin** [reply]
There is an alternative package: https://github.com/rokmohar/medusa-plugin-meilisearch
